tests/statespace/models/structural/components/test_seasonality.py

```diff
@@ -137,20 +146,24 @@ def test_time_seasonality_multiple_observed(rng, remove_first_state):
         np.testing.assert_allclose(matrix, expected)
 
 
-def test_add_two_time_seasonality_different_observed(rng):
+@pytest.mark.parametrize("d1", [1, 2, 3])
+@pytest.mark.parametrize("d2", [1, 2, 3])
```

Comment:
```suggestion
@pytest.mark.parametrize("d1, d2", [(1, 1),  (1, 3), (3, 1), (3, 3)])
```

Our CI runtime is already pretty hefty

Reply:
Thanks for the suggestion. Done.

Reply:
@jessegrabowski No thank you for the support.